coalib/core/Core.py

```diff
@@ -212,8 +212,8 @@ def _schedule_bears(self, bears):
         """
         for bear in bears:
             if self.dependency_tracker.get_dependencies(
-                    bear):  # pragma: no cover
-                logging.warning(
+                    bear):
+                raise RuntimeError(  # Unreachable code
```

Comment:
What "following cleanups".

The current logic will keep iterating on the for loop.  Possibly cleaning up other bears?

So I should defer the raise until after the loop?